plugins/modules/sysrc.py

```diff
@@ -130,7 +130,7 @@ def exists(self):
         Use this dictionary to preform the tests.
         """
         (rc, out, err) = self.run_sysrc('-e', '-a')
-        conf = dict([i.split('=') for i in out.splitlines()])
+        conf = dict([i.split(sep='=', maxsplit=1) for i in out.splitlines()])
```

Comment:
The keywords only work with Python 3, Python 2 only accepts positional arguments:
```
>>> i = "a=1=2"
>>> i.split(sep="=", maxsplit=1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: split() takes no keyword arguments
```
```suggestion
        conf = dict([i.split('=', 1) for i in out.splitlines()])
```